README.md

```diff
@@ -1 +1 @@
-https://github.com/orgs/testbrew/testbrew
+[TestBrew Repo](https://github.com/orgs/testbrew/testbrew)
```

Comment:
https://github.com/orgs/testbrew/testbrew  -> https://github.com/testbrew/testbrew/

Reply:
it's not an org account.

Reply:
lol sorry, that should do it.